drivers/SmartThings/zwave-sensor/src/aeotec-multisensor/multisensor-6/init.lua

```diff
@@ -18,8 +18,20 @@ local cc = require "st.zwave.CommandClass"
 --- @type st.zwave.CommandClass.Configuration
 local Configuration = (require "st.zwave.CommandClass.Configuration")({ version = 2 })
 
+local preferences = require "preferences" 
+
 local MULTISENSOR_6_PRODUCT_ID = 0x0064
 
+local function device_added(self, device)
+  device:send(Configuration:Get({parameter_number = 9})) 
+  device:refresh()
+end
+
+local function device_init(self, device)
+  device:set_update_preferences_fn(preferences.update_preferences)
+  device:send(Configuration:Get({parameter_number = 9}))
+end
```

Comment:
added is correct, but here what I would do is set the update preferences function to a new function that is a composition of the standard preferences update function + the sending of the configuration get